dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -71,6 +70,11 @@ class DefaultEnrollmentService
 
   private final EnrollmentOperationParamsMapper paramsMapper;
 
+  @Override
+  public Enrollment getEnrollment(String uid) throws ForbiddenException, NotFoundException {
```

Comment:
Can you please use our UID type like in the EventService. If its easier you can do it in a separate PR.

Reply:
I'll do it in the following PR